src/main/java/uk/gov/hmcts/reform/coh/controller/DeadlineController.java

```diff
@@ -57,14 +60,28 @@ public ResponseEntity requestExtensionForQuestion(@PathVariable UUID onlineHeari
         }
 
         try {
-            questionService.requestDeadlineExtension(optionalOnlineHearing.get());
```

Comment:
Check the question round is in 'question_issued' state before allowing a deadline extension

Reply:
That's fixed